vendor/github.com/containers/image/docker/tarfile/src.go

```diff
@@ -146,22 +146,22 @@ func (s *Source) ensureCachedDataIsPresent() error {
 	}
```

Comment:
@rhatdan has already done it https://github.com/kubernetes-incubator/cri-o/pull/679